### ISSUE TITLE
Data Structures In Python

### ISSUE THREAD

**Issue by aditya1292** [aditya1292/Data-Structure-In-Python#1]
I am maintaining a DSA repository in Python. If you are a coding freak and want to contribute your best python data structure programs with us then you are welcome to contribute in my dsa repository.

**Comment by KArtik-34** [comment]
Hi. We can add basic Operations on Stacks and Queue (List Implementation ) and Their Basic Idea @aditya1292

**Comment by aditya1292** [comment]
> Hi. We can add basic Operations on Stacks and Queue (List Implementation ) and Their Basic Idea @aditya1292

Yes, but do mention the resources from where you have learnt them all.
